tests/data/TL_alb_0.json

```diff
@@ -0,0 +1,107 @@
+{
+  "MODTRAN": [
```

Comment:
It might be worthwhile to read and write the json files back out using the library. This should make the spacings more consistent.

Reply:
json files written back using the library has been included in the changes in 'modtran_profile_json.py' now